### ISSUE TITLE
Hashtag Strategy - LinkedIn

### ISSUE THREAD

**Issue by c0ric0ri** [Giveth/giveth-planning#1325]
Create a spreadsheet so we can build a list of ideal hashtags to be used for LinkedIn posts. We need to research them and document:

 - Number of followers per hashtag
 - Relevance to our content
 - Which ones to use for what types of posts

**Comment by clara-zi** [comment]
https://docs.google.com/spreadsheets/d/1kV7qXlrx8BAkpv-lukHWgj5I8veqZZcd3ArrLkUM3ZI/edit?usp=sharing

started adding #'s here

**Comment by clara-zi** [comment]
Something I discovered whilst researching hashtags- 'collaborative articles'. Seems to be a new thing.

LinkedIn selects LI members who they consider to have insights to contribute directly to articles within certain topics. It seems I can contribute and add directly to the topic of philanthropy. Here's an example of one of these articles:

<img width="360" alt="image" src="https://user-images.githubusercontent.com/100724132/232798210-8871f855-ecd0-4667-8143-863f3531645d.png">

Here's more info on collaborative article contributions.

https://www.linkedin.com/help/linkedin/answer/a1413111

There are 28k followers of the collaborative article section for philanthropy. Maybe we can use this for Giveth.

**Comment by c0ric0ri** [comment]
Thanks @clara-zi ! This is a great start. 

Here's an article on hashtag strategy for LI: https://blog.hootsuite.com/linkedin-hashtags-guide/

TL;DR

1. 2-5 relevant hashtags per post (Linkedin recommends 3 max, actual max is 10). Post a mix of:
- 1-2 general popular tags (larger followings)
- 1-2 specific niche tags (who we really specifically want to target)

2. Tags should be grouped at bottom of post (looks better out of the way)

3. Capitalize the words to make it look nicer

**Comment by c0ric0ri** [comment]
Moving this to done, FYI @oyealmond
